libs/community/langchain_community/vectorstores/huaweicloud_vector_search.py

```diff
@@ -0,0 +1,487 @@
+import logging
+import uuid
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    Optional,
+    Tuple,
+    Union,
+)
+
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+from langchain_core.vectorstores import VectorStore
+
+if TYPE_CHECKING:
+    from elasticsearch import Elasticsearch
+
+logger = logging.getLogger(__name__)
+
+
+class CSSVectorStore(VectorStore):
```

Comment:
can we call this something like HuaweiElasticsearchVectorStore? CSS usually means something unrelated to vector stores

Reply:
can we call is HuaweiCSSVectorStore? not obvious to an average user that this would refer to a huawei service

Reply:
thanks for your advice, I've renamed it